### PR TITLE
[Benchmark] Add KPI of max, min, avg and p90 latency

### PR DESCRIPTION
**Pull request by JunxiChhen**

### CODE REVIEW

File: benchmark/benchmark.py [hunk 1]
```diff
@@ -174,6 +178,10 @@ def build_inputs_baichuan(tokenizer, query: List[str], padding, history: List[Tu
         print("=" * 50 + args.model_name + " Final Performance" + "=" * 50)
         print(f"Inference Latency:\t{inference_latency:.2f} s")
         print(f"First token Latency:\t{first_token_latency:.2f} ms")
```
Comment (changqi1) [inline benchmark/benchmark.py:180]: print(f"First token Avg Latency:\t{first_token_latency:.2f} ms")

File: benchmark/benchmark.py [hunk 2]
```diff
@@ -163,7 +164,10 @@ def build_inputs_baichuan(tokenizer, query: List[str], padding, history: List[Tu
         output_token_nums = int(torch.numel(generated_ids) / args.batch_size) - input_token_nums
         # Sort the execution times in ascending order
         remained_token_times.sort()
-        # Get the 90th element (index 89) from the sorted list
+        # Get the max, min, avg and 90th element (index 89) from the sorted list
+        latency_max = remained_token_times[-1] * 1000 / (output_token_nums - 1)
```
Comment (changqi1) [inline benchmark/benchmark.py:168]: next_token_latency_max
next_token_latency_min
next_token_latency_max
next_token_latency_90

File: benchmark/benchmark.py (hunk 1 above)
Comment (JunxiChhen) [reply benchmark/benchmark.py]: > print(f"First token Avg Latency:\t{first_token_latency:.2f} ms")

Updated

File: benchmark/benchmark.py (hunk 2 above)
Comment (JunxiChhen) [reply benchmark/benchmark.py]: > next_token_latency_max next_token_latency_min next_token_latency_max next_token_latency_90

Updated

**Review by changqi1 — APPROVED** [review]

**Pull request merged**